paddle/fluid/operators/detection/box_coder_op.cu

```diff
@@ -114,6 +114,22 @@ class BoxCoderCUDAKernel : public framework::OpKernel<T> {
     auto* target_box = context.Input<framework::LoDTensor>("TargetBox");
     auto* output_box = context.Output<framework::Tensor>("OutputBox");
 
+    const T* prior_box_data = prior_box->data<T>();
+    const T* target_box_data = target_box->data<T>();
+    const T* prior_box_var_data;
+    if (!prior_box_var) {
+      framework::Tensor default_box_var;
+      default_box_var.mutable_data<T>(
+          framework::make_ddim({prior_box->dims()[0], prior_box->dims()[1]}),
+          context.GetPlace());
+      math::SetConstant<DeviceContext, T>()(
+          context.template device_context<DeviceContext>(), &default_box_var,
+          static_cast<T>(1));
+      prior_box_var_data = default_box_var.data<T>();
+    } else {
+      prior_box_var_data = prior_box_var->data<T>();
+    }
```

Comment:
We can make `EncodeCenterSizeKernel` and `DecodeCenterSizeKernel` to support whether to consider the variance, instead of creating a constant Tensor with all 1 here.